crates/turbo-malloc/Cargo.toml

```diff
@@ -9,8 +9,5 @@ autobenches = false
 [lib]
 bench = false
 
-[target.'cfg(not(target_os = "linux"))'.dependencies]
-mimalloc-rust = { version = "0.2" }
-
-[target.'cfg(all(target_os = "linux", not(target_arch = "aarch64")))'.dependencies]
-mimalloc-rust = { version = "0.2", features = ["local-dynamic-tls"] }
```

Comment:
This was intentional, as it broke next-swc target platforms. (Including above line 12)